test/windows/event_log/logstash_event_log_plugin_integration.ps1

```diff
@@ -0,0 +1,137 @@
+# Created By: Gabriel Moskovicz
+# 
+# This is a script to test integration between logstash and elasticsearch.
+# It uses a simple json filter to parse the content of a simple text file an then
+# verifying if the message has been found in elasticsearch
+#
+# Requirements to run the test:
+# 
+# - Powershell 4
+# - Windows 7 or newer
+# - Java 7 or newer
+
+Add-Type -assembly "system.io.compression.filesystem"
+
+
+$Main_path = "C:\integration_test"
+If (Test-Path $Main_path){
+	ri -Recurse -Force $Main_path
+}
+$Download_path  = "$Main_path\download"
+md -Path $Download_path
+
+## Logstash variables
+
+$LS_CONFIG="test.conf"
+$LS_BRANCH="2.0"
+$Logstash_path = "$Main_path\logstash"
+$Logstash_zip_file = "$Download_path\logstash.zip"
+$Logstas_URL = "https://s3-eu-west-1.amazonaws.com/build-eu.elasticsearch.org/logstash/$LS_BRANCH/nightly/JDK7/logstash-latest-SNAPSHOT.zip"
```

Comment:
I see some of the code is the same as `test/windows/acceptance/logstash_release_acceptance.ps1`
Can we refactor it to remove the duplication?